react/src/utils/tableHelpers.ts

```diff
@@ -108,3 +111,91 @@ export const sortQueryResult = (queryResult: VariantQueryDataResult[]) => {
     );
     return sortedQueryResult;
 };
+
+export const isHeterozygous = (zygosity: String | null | undefined) => {
+    if (zygosity?.toLowerCase().includes('het')) {
+        return true;
+    }
+    return false;
```

Comment:
This and the following line can be just one-line if you do `return !!zygosity?.toLowerCase().includes('het')` to get rid of the nullish values.